File: week6/Easy/IntWrapper.java
public class IntWrapper {

    private int value;
    
    public IntWrapper(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

    public void doubleValue() {
        this.value *= 2;
    }

}

File: week6/Easy/int_wrapper.py
class IntWrapper:
    def __init__(self, value):
        self.value = value
    
    def get_value(self):
        return self.value
    
    def set_value(self, value):
        self.value = value
    
    def double_value(self):
        self.value *= 2


if __name__ == '__main__':
    iw = IntWrapper(3)
    print(iw.get_value())

    iw.set_value(5)
    print(iw.get_value())

    iw.double_value()
    print(iw.get_value())

